### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by kestewart**
Add in contributors to the spec over the last year.

### CODE REVIEW

**Comment by kestewart** [comment]
@goneall - I've added in Meret & Armin.   Let me know if you spot any others that should be added.

File: docs/index.md [hunk 1]
```diff
@@ -30,18 +33,21 @@ Eran Strod,
...
 Henk Birkholz,
 Hiroyuki Fukuchi,
 Jack Manbeck,
 Jaime Garcia,
+Jeff Liquia,
 Jeff Luszcz,
 Jeff Schutt,
 Jilayne Lovejoy,
 John Ellis,
 Jonas Oberg,
```
Comment (goneall) [inline docs/index.md:48]:
```suggestion
Jonas Oberg,
Joshua Watt,
```

File: docs/index.md [hunk 2]
```diff
@@ -9,10 +9,13 @@ Copies of these licenses are reproduced in their entirety herein.
...
+Adolfo García Veytia,
 Alan Tse,
 Alexios Zavras,
 Andrew Back,
 Ann Thornton,
+Armin Tänzer,
+Arthit Suriyawongkul,
 Basil Peace,
 Bill Schineller,
 Bradlee Edmondson,
```
Comment (goneall) [inline docs/index.md:21]:
```suggestion
Bradlee Edmondson,
Brandon Lum,
```

**Review by goneall — CHANGES_REQUESTED** [review]
A couple suggested addes.

Also, please add the following (Github would not allow me to make this as a suggestion):
Takashi Ninjouji

**Review by goneall — APPROVED** [review]
LGTM

**Review by puerco — APPROVED** [review]
LGTM :two_hearts:

**Comment by kestewart** [comment]
Given we've got 2 +1's going ahead and merging this.

**Pull request merged**